colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Y'CbCr Colour Encoding
+======================
+
+Defines the Y'CbCr encoding (Y'CbCr is not a colour space) transformations:
```

Comment:
We tend to write **colour space** as **colourspace** in `colour`, that being said, I have checked in ISO 22028-1, and they use **colour space**. I would just change it for consistency, we can debate about the proper term later :)